portfolio/src/main/java/com/google/sps/servlets/BackgroundChange.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
```

Comment:
This is duplicated from #2 I think. For the sake of clarity, could you merge that PR, rebase, and then send for review?